client/src/extension.ts

```diff
@@ -52,10 +50,22 @@ export async function activate (context: ExtensionContext): Promise<void> {
   // Create the language client and start the client.
   client = new LanguageClient('bitbake', 'Language Server Bitbake', serverOptions, clientOptions)
 
+  client.onRequest('custom/verifyConfigurationFileAssociation', async (param) => {

```

Comment:
In my latest work, I tried to keep extension.ts as tidy as possible. It's now starting to have a lot of features so it needs structure IMO. I suggest to move this code in the src/language folder to not bloat the activate() function.

Reply:
sure, I just know your PR was merged. I can rebase now